remo/events/migrations/0022_remove_duplicate_action_items.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+# flake8: noqa
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """Delete duplicate action items for post event metrics.
+
+        This affects only the events with end date after the 1st of May 2015.
+        """
+        from remo.dashboard.models import ActionItem
+
+        event_type = orm['contenttypes.ContentType'].objects.get(
+            app_label='events', model='Event')
+        end = datetime.datetime.combine(datetime.date(2015, 5, 1),
+                                        datetime.datetime.min.time())
+        events = orm['events.Event'].objects.filter(end__gt=end)
+
+        for event in events:
+            action_item_q = {'content_type': event_type,
+                             'object_id': event.id,
+                             'priority': ActionItem.NORMAL,
+                             'user': event.owner}
+            action_items = orm['dashboard.ActionItem'].objects.filter(
+                **action_item_q).order_by('id').values_list('id', flat=True)
```

Comment:
Since we are doing a bulk deletion, I guess it makes sense to keep the older action item.